x/docs/md/help/executable.md

```diff
@@ -28,10 +28,10 @@ If the API key is incorrect, then re-run the `configure` command:
 exercism configure --key=YOUR_API_KEY
 ```
 
-Next look at the `Exercises Directory`. The current directory which is being used to fetch exercises can be viewed using the following command:
+Next look at the Exercises Directory. It will look something like this:
 
-```bash
-exercism configure --dir
+```plain
+Exercises Directory: /home/you/exercism
```

Comment:
Ooh. Wait. The `exercism configure` output looks like this:

```

Configuration written to /Users/kytrinyx/.exercism.json

  --key=<redacted>
  --dir=/Users/kytrinyx/tmp/exercism
  --host=http://exercism.io
  --api=http://x.exercism.io

```

So look for `--dir`, rather than `Exercises directory`. There's less output, which makes it easier to scan than debug.